React/app/containers/App/index.js

```diff
@@ -30,6 +31,7 @@ export default function App() {
         <Route path="/donors/:id" component={DonorHomePage} />
         <Route path="/volunteers/:id" component={MemberHomePage} />
         <Route path="/account/:id" component={AccountPage} />
+        <Route path="/charity" component={CharityDetailPage} />
```

Comment:
One thing to think about is how will we access a charity. If you want to try and do what I did with '/:id' ending look up 'React useParams' and you can pull the id from the url then. (that would allow us to do a fetch request, with axios, on componentDidMount func). Just look up axios and fetch, along with the componentDidMount func for React. This will be some good material to checkout that will help you writing containers.